thirdparty/kyaml/runfn/runfn.go

```diff
@@ -386,23 +386,24 @@ func getUIDGID(asCurrentUser bool, currentUser currentUserFunc) (string, error)
 	return fmt.Sprintf("%s:%s", u.Uid, u.Gid), nil
 }
 
-// etFunctionConfig returns yaml representation of functionConfig that can
+// getFunctionConfig returns yaml representation of functionConfig that can
 // be provided to a function as input.
 func (r *RunFns) getFunctionConfig() (*yaml.RNode, error) {
```

Comment:
can this function be moved to fnruntime/runner.go and shared between `eval` and `render`? What's the reason for keeping them separate?